shoop_tests/simple_pricing/test_simple_pricing.py

```diff
@@ -6,46 +6,138 @@
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
 import pytest
-from shoop.core.pricing.price import TaxfulPrice
-from shoop.testing.factories import get_default_shop, create_product
+from shoop.core import settings
+from shoop.core.pricing.price import TaxfulPrice, TaxlessPrice
+from shoop.testing.factories import get_default_shop, create_product, get_default_customer_group, create_random_person
 from shoop.simple_pricing.module import SimplePricingModule
 from shoop.simple_pricing.models import SimpleProductPrice
 
 
-@pytest.mark.django_db
-def test_shop_specific_cheapest_price_1(rf):
+def get_shop_with_tax(include_tax):
     shop = get_default_shop()
+    shop.prices_include_tax = include_tax
+    shop.save()
+    return shop
+
+def initialize_test(rf, include_tax=False):
+    shop = get_shop_with_tax(include_tax=include_tax)
+
+    group = get_default_customer_group()
+    customer = create_random_person()
+    customer.groups.add(group)
+    customer.save()
+
     request = rf.get("/")
     request.shop = shop
-    product = create_product("Just-A-Product")
-    SimpleProductPrice.objects.create(product=product, shop=None, price=200,)
-    SimpleProductPrice.objects.create(product=product, shop=shop, price=250)
+    request.customer = customer
+    return request, shop, group
+
+
+@pytest.mark.django_db
+def test_shop_specific_cheapest_price_1(rf):
+    request, shop, group = initialize_test(rf, False)
+
+    product = create_product("Just-A-Product", shop, default_price=200)
+
+    # determine which is the taxfulness
+    price_cls = TaxfulPrice if settings.SHOOP_DEFAULT_PRICES_INCLUDE_TAX else TaxlessPrice
```

Comment:
Shouldn't this use the Shop.prices_include_tax instead? (and the setting be removed)